<launch>
    <node name="data_publisher" pkg="exoskeleton" type="data.py"/>
</launch>

